3-enrich/README.md

```diff
@@ -15,6 +15,6 @@ They are both part on [enrich][enrich] project.
 
 [storage]: https://github.com/snowplow/snowplow/tree/master/4-storage
 
-[docs-beam-enrich]: https://docs.snowplowanalytics.com/docs/setup-snowplow-on-gcp/setup-validation-and-enrich-beam-enrich/
-[docs-stream-enrich]: https://docs.snowplowanalytics.com/docs/setup-snowplow-on-aws/setup-validation-and-enrich/
+[docs-beam-enrich]: https://docs.snowplow.io/docs/getting-started-on-snowplow-open-source/setup-snowplow-on-gcp/setup-validation-and-enrich/
```

Comment:
I would also fix the link names, not that it changes the resulting text, but could be confusing when we edit it.

Reply:
(It should not say `beam-enrich`)

Reply:
Actually hold on, we need to fix the text as well! It should be enrich-kinesis and enrich-pubsub, not stream-enrich and beam-enrich.

Reply:
Oh true, good catch. One sec